constance/admin.py

```diff
@@ -190,7 +191,9 @@ def changelist_view(self, request, extra_context=None):
                 return HttpResponseRedirect('.')
         context = {
             'config_values': [],
-            'title': _('Constance config'),
+            'title': _('{0} {1}'.format(
+                self.model._meta.app_config.verbose_name.title(),
+                self.model._meta.verbose_name_plural)),
             'app_label': 'constance',
```

Comment:
A bunch of things are wrong here:

- Please don't use `title()` on the verbose name, no need as this may end up being wrong in languages where words aren't capitalized.
- Don't use string formatting to create a string that is then marked for translation, it won't be picked up by older versions of gettext or simply offered as a string with two unnamed variables, hardly descriptive for translators.
- Don't concatenate two different strings that are marked for translation ("Constance" and "config") since in many languages that particular order of words may not make sense.

Instead use this for the title context variable: `self.model._meta.app_config.verbose_name`